### PR TITLE
Change "Multiple" to "Mixed"

### LINKED ISSUE #2088: Change wording of multi-value input elements
## Feature Description

When a selection includes multiple values for a design panel input element (e.g. font or color), currently the text "Multiple" is displayed (in a few variants). These should all be changed to "Mixed" consistently through all the panels.

Note that this might even occur with a single element selection, as text fields can have e.g. multiple font weights or colors inside them.

## Alternatives Considered

* None

## Additional Context

* From [UX Evaluation](https://docs.google.com/presentation/d/1ZiZWrmLiX40rt3n_1LMSGJxYWI-xsWFsS9bkzdbRIV8/edit?ts=5ec35b92#slide=id.g77fdc779d5_0_187)

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Implementation Brief

* Please create a comprehensive lists of components to be changed here and get sign-off from another dev before starting - just to make sure all relevant components have been identified
* Make sure value is translatable in all locations
* After this, create unit tests for all of these components to display the correct mixed value when applicable - even for components not yet tested

List of components to be changed:
- `Numeric`
- `ColorPreview`
- `TextInput`
- Font family
- Font weight

### PR DESCRIPTION
**Pull request by miina**
## To-do

- [x] Tests

## Testing Instructions
1. Add two elements of the same type (try with Text, Shape, Image, Video)
2. Set all the input values so that they would be different for the two elements.
3. Select the two elements.
4. Verify that "Mixed" is used in the inputs whenever the two values differ as opposed to "Multiple"

Note: There is a bug for Text elements where one element doesn't have a background and the other has -- in this case "Mixed" it not shown and the background of the element with the background is shown instead. Please ignore that when testing.

---

Fixes #2088

### CODE REVIEW

File: assets/src/edit-story/components/richText/karma/multiSelection.karma.js [hunk 1]
```diff
@@ -146,8 +146,8 @@ describe('Styling multiple text fields', () => {
       expect(underline.checked).toBe(false);
       expect(fontWeight.value).toBe('(multiple)');
```
Comment (miina) [inline assets/src/edit-story/components/richText/karma/multiSelection.karma.js]: Oops, this still needs  👀

**Review by barklund — APPROVED** [review]
This works excellent.

I tried to make to text fields completely different in every way, but one weird thing happened (clickbait!):

Screen Shot 2020-10-11 at 12 24 44

As only one of the text fields has a background color, that color is listed as the background color in both text fields. That seems wrong somehow though?

**Edit**: I know this is not directly related to this PR, so it could be also just be filed as a new bug.

Other than that, it looks good with _"Mixed"_ all over - it's a better word than _"Multiple"_ (which was simply a legacy from my very first original PoC, I think).

**Comment by barklund** [comment]
> As only one of the text fields has a background color, that color is listed as the background color in both text fields. That seems wrong somehow though?
> 
> **Edit**: I know this is not directly related to this PR, so it could be also just be filed as a new bug.

I now see bug #2338, which kind of capture this, so maybe we don't need to do anything more about this at this point. It's a minor bug, so it's fine to just have it filed.

**Comment by miina** [comment]
> I now see bug #2338, which kind of capture this, so maybe we don't need to do anything more about this at this point. It's a minor bug, so it's fine to just have it filed.

Addressing #2338 should indeed fix this issue as well. Will send to QA as-is.

**Comment by csossi** [comment]
Verified in QA

**Pull request merged**